syserol/classify.py

```diff
@@ -43,6 +43,30 @@ def SVM_2class_predictions(subjects_matrix):
     return cp_accuracy, nv_accuracy
 
 
+def logistic_2class_predictions(subjects_matrix):
+    """ Predict Subject Class with Decomposed Tensor Data and Logistic Regression"""
+    # Load Data
+    classes = load_file("meta-subjects")
+    classes = classes.replace(
+        to_replace=["controller", "progressor", "viremic", "nonviremic"],
+        value=[1, 0, 1, 0],
+    )
+    cp = np.array(classes["class.cp"])
+    nv = np.array(classes["class.nv"])
+
+    # Controller/Progressor classification
+    Y = cp
+    Y_pred1 = cross_val_predict(LogisticRegressionCV(), subjects_matrix, Y)
```

Comment:
Just like with elastic net, you should use LogisticRegressionCV to determine the regularization, then LogisticRegression for the actual cross val / prediction.

Reply:
In terms of determining the regularization, LogisticRegressionCV has the attribute Cs_, which give the inverse regularization parameter values. But LogisticRegression only takes in one value for C. Is there a different value we should be taking from LogisticRegressionCV to put into LogisticRegression? Or should we just run LogisticRegression without any input parameters? I guess I am just a bit confused about how running LogisticRegressionCV before running LogisticRegression in a CV will help if we don't actually take any parameters from the first LRCV run?

Reply:
Use `C_` and `l1_ratio_` from LogisticRegressionCV to set `C` and `l1_ratio` in LogisticRegression. Both should also use `penalty="elasticnet"`.

Reply:
It's currently throwing an error when I try to fit LogisticRegressionCV. I think it may want an input value for `l1_ratios` since we've set it to `penalty="elasticnet`. Do you have a value you want inputted for the ratio?

Reply:
You can use `l1_ratios=[0.0, 0.1, 0.5, 0.9, 1.0]`